src/pages/Reserve/Reservation.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Loading from '../../components/Buttons/Loading';
+import HouseDetail from '../../components/Houses/HouseDetail';
+import getHouses from '../../redux/actions/House/getHouses';
+
+function Reservation() {
```

Comment:
- [ ] I can reserve houses, however, each time I reserve a house and get redirected to "my reservations", I am greeted by a blank page.  Kindly fix this behavior so your app works as intended. 💪